paddle/fluid/operators/elementwise_op_function.h

```diff
@@ -468,15 +483,22 @@ static __global__ void ElemwiseGradBroadcast2CUDAKernel(
   }
 
   if (dy) {
+    T val = shm[threadIdx.x];
+    int warpSize = 32;
+    for (int offset = warpSize / 2; offset > 0; offset /= 2)
+      val += __shfl_down_sync(0, val, offset);
```

Comment:
The first parameter of __shfl_down_sync() is a mask. It may be `-1UL`.

Reply:
Yes, but the first parameter of __shfl_down_sync() is invalid when CUDA_VERSION is less 9000.